composer.json

```
@@ -14,10 +14,10 @@
         "php": "^7.1",
         "ext-json": "*",
         "elasticsearch/elasticsearch": "^7.0",
-        "laravel/framework": "^5.7"
+        "laravel/framework": "^5.7|5.8.*|^6.0|^7.0"
     },
     "require-dev": {
-      "phpunit/phpunit": "^7.1",
+    "phpunit/phpunit": "^7.1",
```

You seem to have removed the tab?
```suggestion
        "phpunit/phpunit": "^7.1",
```